tests/ballerina-compiler-api-test/src/test/resources/test-src/symbols/clients_decl_symbol_test.bal

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2021 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+client "http://example.com" as cl1;
```

Comment:
Shall we test a client declaration inside a statement-block as well (Or we don't need to test that scenario?)